addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/handler/NetworkHandler.java

```diff
@@ -50,20 +52,13 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         if (command instanceof RefreshType) {
             switch (channelUID.getId()) {
                 case CHANNEL_ONLINE:
-                    try {
-                        State state = networkService.updateDeviceState() < 0 ? OnOffType.OFF : OnOffType.ON;
-                        updateState(CHANNEL_ONLINE, state);
-                    } catch (InvalidConfigurationException invalidConfigurationException) {
-                        updateStatus(ThingStatus.OFFLINE);
-                    }
+                    // The GUI requested a refresh. We send the cached value immediately and reset the refresh timer to
+                    // get the device reachable state as fast as possible.
```

Comment:
So what is the change in #2380 doing then? I still think that messing with the scheduled jobs here is not a good idea. If it isn't done, the user will only see more frequent checks and not less, which isn't a bad thing. And it would be in line with all other bindings that work with a polling interval.